components/file_attachment/file_attachment.test.tsx

```diff
@@ -16,118 +16,130 @@ jest.mock('utils/utils.jsx', () => {
     };
 });
 
-function createComponent({fileInfo, handleImageClick, index, compactDisplay, canDownloadFiles = true, enableSVGs = false} = {}) { //eslint-disable-line react/prop-types
-    const fileInfoProp = fileInfo || {
-        id: 1,
+describe('FileAttachment', () => {
+    const baseFileInfo = {
+        id: 'thumbnail_id',
         extension: 'pdf',
         name: 'test.pdf',
         size: 100,
+        width: 100,
+        height: 80,
+        has_preview_image: true,
+        user_id: '',
+        create_at: 0,
+        update_at: 0,
+        delete_at: 0,
+        mime_type: '',
+        clientId: '',
+    };
+
+    const baseProps = {
+        fileInfo: baseFileInfo,
+        handleImageClick: jest.fn(),
+        index: 3,
+        canDownloadFiles: true,
+        enableSVGs: false
```

Comment:
I hate to be a stickler but can you add the comma here too?

Reply:
Sure, and will try to tune my local config to make eslint warn me about this.